docs/pages/workflow/configuration.md

```diff
@@ -54,7 +54,7 @@ export default {
 Extras can be accessed via `expo-constants`:
 
 ```ts
-// App.js
+// App.json
```

Comment:
The original annotation was correct, the code snippet is from app.js, not app.json.